exasol_transformers_extension/udfs/sequence_classification_single_text_udf.py

```diff
@@ -0,0 +1,44 @@
+import torch
+import transformers
+from exasol_transformers_extension.udfs import bucketfs_operations
+
+
+class SequenceClassificationSingleText:
+    def __init__(self,
+                 exa,
+                 cache_dir=None,
+                 base_model=transformers.AutoModelForSequenceClassification,
+                 tokenizer=transformers.AutoTokenizer):
+        self.exa = exa
+        self.cache_dir = cache_dir
+        self.base_model = base_model
+        self.tokenizer = tokenizer
+
+    def run(self, ctx):
+        bucketfs_conn = ctx.bucketfs_conn
+        text = ctx.text
+        model_name = ctx.model_name
+
+        # get bucketfs location
+        bucketfs_conn_obj = self.exa.get_connection(bucketfs_conn)
+        bucketfs_location = bucketfs_operations.create_bucketfs_location(
+            bucketfs_conn_obj)
+
+        # get cache directory
```

Comment:
if  the cache_dir is not given, we shouldn't fail because we get it from model_name

Reply:
This means we can remove the cache_dir, because in the downloader we expect the bucketfs connection gives us the cache_dir root, and we then store the model under its name. Alternatively, we use it in DownloaderUDF and PredictionUDF, to allow the user to specify subdirectories for the bucketfs con. The latter is probably a good idea, because the user might get a single connection from the admin and needs to use it for different things.